src/RSVP.tsx

```diff
@@ -0,0 +1,173 @@
+import { useState } from "react";
+import { XMarkIcon } from "@heroicons/react/16/solid";
+import { Input, Button } from "@heroui/react";
+
+interface RSVPProps {
+  event: {
+    title: string;
+    description: string;
+    location: string;
+    sDate: string;
+    creatorEmail: string;
+  };
+}
+
+export default function RSVP({ event }: RSVPProps) {
+  const [rsvpResponse, setRsvpResponse] = useState<"Yes" | "No" | "Maybe" | null>(null);
+  const [rsvpSubmitted, setRsvpSubmitted] = useState(false);
+  const [showRsvpModal, setShowRsvpModal] = useState(false);
+  const [pendingResponse, setPendingResponse] = useState<"Yes" | "No" | "Maybe" | null>(null);
+  const [userName, setUserName] = useState("");
+
+  const handleRsvpClick = (response: "Yes" | "No" | "Maybe") => {
+    setPendingResponse(response);
+    setShowRsvpModal(true);
+  };
+
+  const handleRsvpSubmit = () => {
+    if (!pendingResponse || !event.creatorEmail) return;
+
+    const name = userName.trim() || "Someone";
+    const responseText = pendingResponse;
+    
+    // Create mailto link
+    const subject = `RSVP: ${responseText} - ${event.title}`;
+    const body = `Hi,
+
+${name} is responding to your event: "${event.title}"
+
+Response: ${responseText}
+
+Event Details:
+${event.description ? `Description: ${event.description}` : ''}
+${event.location ? `Location: ${event.location}` : ''}
+${event.sDate ? `Date: ${event.sDate}` : ''}
+
+Best regards,
+${name}`;
```

Comment:
Put the poor calf somewhere at the bottom:

Something like this is enough:
❤️ [Calf](https://omid.github.io/calf/) 🐮📅